datasets/br/br_tcu_disqualified/crawler.py

```diff
@@ -0,0 +1,92 @@
+from typing import List
+from zavod import Context, helpers as h
+
+
+def fetch_data(context: Context) -> List[dict]:
+    """
+    Fetches data from the website, or raises an exception on failure.
+
+    :param context: The context object.
+    :return: List of dicts containing data.
+    """
+    response = context.fetch_json(context.data_url)
+    if "items" not in response:
+        context.log.error("Items not found in JSON")
+        return []
+    return response["items"]
```

Comment:
you can probably just fold this into crawl() - it's not being reused I don't think we're gaining from it being a separate function.